src/rspec/SpecParser.ts

```diff
@@ -6,7 +6,7 @@ interface SpecRegion {
 };
 
 export class SpecParser {
-  private matcher = /(?:(?:^\s*(?:RSpec\.)?(?:it|context|example)\(?\s*(?:(['"])(?<title>.*)\1)?\)?\s*(?:,\s*[\w:'"]+\s*(?:=>)?\s*['":\w]+)?\s*(?:do|{))|(?:^\s*(?:it_behaves_like|include_examples)\s*(['"])(?:.*)\3\s*(?:do|{)?)|(?:^\s*(?:RSpec\.)?describe\(?\s*(?:(?<describedClass>[\w:\.]+)|(?:(['"])(?<describedString>.*)\5))\)?\s*(?:,\s*:?['"\w]+\s*(?:=>|:)\s*['"\w:]+\s*)*\s*(?:do|{))|(?:^\s*(?:it|example)\s*{(?<singleLine>.*)}))\s*(?:#.*)?$/;
+  private matcher = /(?:(?:^\s*(?:RSpec\.)?(?:it|context|example|specify|scenario)\(?\s*(?:(['"])(?<title>.*)\1)?\)?\s*(?:,\s*[\w:'"]+\s*(?:=>)?\s*['":\w]+)?\s*(?:do|{))|(?:^\s*(?:it_behaves_like|include_examples)\s*(['"])(?:.*)\3\s*(?:do|{)?)|(?:^\s*(?:RSpec\.)?describe\(?\s*(?:(?<describedClass>[\w:\.]+)|(?:(['"])(?<describedString>.*)\5))\)?\s*(?:,\s*:?['"\w]+\s*(?:=>|:)\s*['"\w:]+\s*)*\s*(?:do|{))|(?:^\s*(?:it|example)\s*{(?<singleLine>.*)}))\s*(?:#.*)?$/;
```

Comment:
```suggestion
  private matcher = /(?:(?:^\s*(?:RSpec\.)?(?:it|context|example|specify|scenario)\(?\s*(?:(['"])(?<title>.*)\1)?\)?\s*(?:,\s*[\w:'"]+\s*(?:=>)?\s*['":\w]+)?\s*(?:do|{))|(?:^\s*(?:it_behaves_like|include_examples)\s*(['"])(?:.*)\3\s*(?:do|{)?)|(?:^\s*(?:RSpec\.)?(?:describe|feature)\(?\s*(?:(?<describedClass>[\w:\.]+)|(?:(['"])(?<describedString>.*)\5))\)?\s*(?:,\s*:?['"\w]+\s*(?:=>|:)\s*['"\w:]+\s*)*\s*(?:do|{))|(?:^\s*(?:it|example|specify|scenario)\s*{(?<singleLine>.*)}))\s*(?:#.*)?$/;
```

For the sake of completeness, if `specify` and `scenario` are aliases for `it` we should probably support the crazy single line syntax as well. I also looked at the capybara spec and discovered that `feature` is an alias for `describe` so we should try and support that too.

Reply:
sounds good!